metadata-ingestion/source_docs/hana.md

```diff
@@ -0,0 +1,88 @@
+# SAP HANA
+
+For context on getting started with ingestion, check out our [metadata ingestion guide](../README.md).
+
+## Setup
+
+To install this plugin, run `pip install 'acryl-datahub[hana]'`.
+
+## Capabilities
+
+This plugin extracts the following:
+
+- Metadata for databases, schemas, and tables
+- Column types and schema associated with each table
+- Table, row, and column statistics via optional [SQL profiling](./sql_profiles.md)
+
+| Capability        | Status | Details                                  | 
+|-------------------|--------|------------------------------------------|
+| Platform Instance | ✔️     | [link](../../docs/platform-instances.md) |
+| Data Containers   | ✔️     |                                          |
+| Data Domains      | ✔️     | [link](../../docs/domains.md)            |
+
+
+## Quickstart recipe
+
+Check out the following recipe to get started with ingestion! See [below](#config-details) for full configuration options.
+
+For general pointers on writing and running a recipe, see our [main recipe guide](../README.md#recipes).
+
+```yml
+source:
+  type: hana
+  config:
+    # Coordinates
+    host_port: localhost:39041
+    database: dbname
+
+    # Credentials
+    username: SYSTEM
+    password: example
```

Comment:
Please don't put dummy passwords as example. This is bad practice that we are trying to remove. Please add something like "${HANA_PASS}"

Reply:
Changed the documentation accordingly. Everything should be resolved